app/assets/stylesheets/db/controllers/draft-multiple-episodes.scss

```diff
@@ -0,0 +1,5 @@
+body.db-draft_multiple_episodes {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `db-draft_multiple_episodes` should be written in lowercase with hyphens